v1/pipeline/steps/custom/service-now-approval-step-info.yaml

```diff
@@ -5,21 +5,21 @@ allOf:
   required:
   - approval_criteria
   - connector
-  - ticket_number
-  - ticket_type
+  - number
+  - type
   properties:
     approval_criteria:
       $ref: criteria-spec-wrapper.yaml
-    change_window:
+    window:
       $ref: service-now-change-window-spec.yaml
     connector:
       type: string
     delegates:
       $ref: ../../common/delegate.yaml
     rejection_criteria:
       $ref: criteria-spec-wrapper.yaml
-    ticket_number:
```

Comment:
ticket: 
  number : string
  type: xyz
  
  wouldn't this make more sense ?

Reply:
Done